examples/40_advanced/example_text_preprocessing.py

```diff
@@ -1,79 +1,98 @@
 # -*- encoding: utf-8 -*-
 """
-==================
-Text Preprocessing
-==================
-This example shows, how to use text features in *auto-sklearn*. *auto-sklearn* can automatically
-encode text features if they are provided as string type in a pandas dataframe.
+==============
+Text preprocessing
+==============
+
+The following example shows how to fit a simple NLP problem with
+*auto-sklearn*.
+
+For deeper insights into the field of text preprocessing you can follow these links:
+    1. https://scikit-learn.org/stable/tutorial/text_analytics/working_with_text_data.html
+    2. https://machinelearningmastery.com/clean-text-machine-learning-python/
 
-For processing text features you need a pandas dataframe and set the desired
-text columns to string and the categorical columns to category.
 
-*auto-sklearn* text embedding creates a bag of words count.
 """
+from pprint import pprint
+
+import pandas as pd
+from sklearn.datasets import fetch_20newsgroups
 import sklearn.metrics
-import sklearn.datasets
+
 import autosklearn.classification
 
 ############################################################################
 # Data Loading
 # ============
 
-X, y = sklearn.datasets.fetch_openml(data_id=40945, return_X_y=True)
-
-# by default, the columns which should be strings are not formatted as such
-print(f"{X.info()}\n")
+newsgroups_train = fetch_20newsgroups(subset="train")
```

Comment:
Would it make sense to use the `remove` argument here to remove columns that make it too easy?

Reply:
Do not know what argument i to remove, because 20 Newsgroups has only 1 column